scripts/deploy.memecoin.mjs

```diff
@@ -52,11 +53,23 @@ const randomMemecoin = {
 
 const ledgerCanisterId = await deployLedger(randomMemecoin);
 
-await deployIndex({
-	...randomMemecoin,
-	ledgerCanisterId
-});
+const args = process.argv.slice(2);
+const withIndex = args?.find((arg) => ['--with-index'].includes(arg)) !== undefined;
```

Comment:
can this be simplified with `(args ?? []).includes('--with-index')` ? or am I missing something?

Reply:
Sure

Reply:
Done